build/cqs.tsv

```diff
@@ -1,19 +1,25 @@
+asm:asm	3.1	compile
 asm:asm	3.3.1	compile
```

Comment:
Re: asm, we have an entry for 3.1, but not 3.3.1.

Can someone comment on how ASM is used?